source/users.rst

```diff
@@ -177,6 +242,7 @@ Request:
     "username": "example",
     "password": "password",
     "email": "example@example.com"
+    "roles": ["intern"],
```

Comment:
If the API as written currently has the client sending the server slugs and the server sending the client names, then that's what should happen here.  I do wish I'd stepped in back when the API was first written to say that that sounds like a bad idea, but it's way too late for that now!